src/h5web/BreadcrumbsBar.tsx

```diff
@@ -38,12 +38,10 @@ function BreadcrumbsBar(props: Props): JSX.Element {
       {selectedNode && (
         <div className={styles.breadCrumbs}>
           {selectedNode?.parents.slice(firstParentIndex).map(member => (
-            <>
-              <span className={styles.crumb} key={(member as HDF5HardLink).id}>
-                {member.title}
-              </span>
+            <Fragment key={(member as HDF5HardLink).id}>
+              <span className={styles.crumb}>{member.title}</span>
               <FiChevronRight />
-            </>
+            </Fragment>
```

Comment:
Just a quick fix for a warning in the console. The `key` has to go on the fragment.